pkg/core/resources/apis/mesh/validators.go

```diff
@@ -225,3 +255,45 @@ func ValidateResourceYAMLPatch(msg proto.Message, resYAML string) error {
 	}
 	return (&jsonpb.Unmarshaler{}).Unmarshal(bytes.NewReader(json), msg)
 }
+
+// SelectorKeyNot returns a TagKeyValidatorFunc that checks the tag key
+// is not keyName.
+func SelectorKeyNot(keyName string) TagKeyValidatorFunc {
```

Comment:
Why do we need this if we always can call `SelectorKeyNotInSet` with a single argument?

Reply:
Yup, I can remove this.